optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerBuildTimeConfig.java

```diff
@@ -31,6 +32,8 @@
 public class OptaPlannerBuildTimeConfig {
 
     public static final String DEFAULT_SOLVER_CONFIG_URL = "solverConfig.xml";
+    public static final String DEFAULT_CONSTRAINTS_DRL_URL = "constraints.drl";
+    public static final String CONSTRAINTS_DRL_PROPERTY = "quarkus.optaplanner.constraints-drl";
```

Comment:
None of the other properties have a hard coded string like this in the non-test classes. Can we move it to the test class?

Reply:
I see your point, however, the property name is used also in exception messages when we fail fast.